### DISCUSSION TITLE
Want to hide crisp chatbox icon which is overlapping torus embed wallet

### DISCUSSION THREAD

**Discussion by sappysealsz** [Web3Auth/Web3Auth#606] [Feature Requests]
Hi @everyone,

I noticed that there is always showing crisp chatbox icon overlapped with torus embed wallet icon after web3auth login
Even it happens in web3auh demo (torus-app)
<img width="1476" alt="Screenshot 2022-09-13 at 16 05 28" src="https://user-images.githubusercontent.com/44206695/189909963-82aaa19d-5ac9-4e42-a26c-a78bf514fe17.png">
It doesn't happen with ether net (when use TorusWalletConnectorPlugin), but happens with solana (when use SolanaWalletConnectorPlugin)
<img width="1490" alt="Screenshot 2022-09-13 at 16 37 01" src="https://user-images.githubusercontent.com/44206695/189915954-523bb47d-1fa1-4073-83fa-c3a00e0bf9f9.png">

I couldn't find any option to hide the crisp chatbox.
I asked this problem in discord channel, and someone mentioned to update web3auth packages with latest version.
I tried it, but didn't help

It would be much appreciated if someone let me know the way to solve this problem.
Thanks

**Comment by sappysealsz** [comment]
I am attaching the source code here.
Hopefully, it would be helpful to figure out the problem.
[torus-app.zip](https://github.com/Web3Auth/Web3Auth/files/9557550/torus-app.zip)

**Reply by rinzler99** [reply]
Hey danny, Thanks for reaching out team will be fixing this issue soon.

**Reply by sappysealsz** [reply]
Thanks, @rinzler99

**Reply by chaitanyapotti** [reply]
Seems to be a bug with crisp in certain regions. it should go away in certain time auto

**Reply by sappysealsz** [reply]
Could you please add an option to hide the crisp chatbox?

**Reply by rinzler99** [reply]
Thanks for your suggestion! But it won't appear though as mentioned it will go away in certian time.

**Reply by sappysealsz** [reply]
The crisp icon is always there. And, I want to remove it totally because I am using intercom chatbox instead of crisp embeded in torus wallet iframe. As it resides in the torus wallet iframe, I couldn't hide it manually using css due to CORS problem.
What's the solution to hide the crisp chatbox?  
Again, I would like to remove the crisp chatbox from our project. It shouldn't displayed on our project in any time, (not matter it should go away in certain time auto)

**Reply by sappysealsz** [reply]
We integrated web3auth into our project and let users use torus embed wallet. But, with this bug, it's inconvenient to use torus embed wallet for our users. So it's an urgent problem on our end.

**Reply by rinzler99** [reply]
Hey, Please give us sometime will update on this soon. Thanks!

**Reply by sappysealsz** [reply]
Cool! Thank you!

**Reply by sappysealsz** [reply]
Hi @chaitanyapotti , @rinzler99 , Is there any update regarding this issue?

**Reply by rinzler99** [reply]
Hey, Our team tried to reproduce and debugged the issue seems like crisp will pop up based on the regions. It will go away automatically.

**Reply by sappysealsz** [reply]
Hi @rinzler99 ,
Thanks for your reply. I think you can reproduce it by clearing browser cache every time when you run the source code I've attached. 
If it still doesn't happen, you can also try with VPN by choosing Russia.
